Idno/Common/Entity.php

```diff
@@ -2670,6 +2671,12 @@ function getAnnotationSubtype($uuid)
          */
         function getAnnotation($uuid)
         {
+            // Prioritize Annotation methods
+            $annotation = Annotation::getByUUID($uuid);
```

Comment:
Overriding the internals here allow us to keep using the same `getAnnotation()` calls used in templates.